src/main/kotlin/net/maizegenetics/phgv2/pathing/ImputationMetrics.kt

```diff
@@ -0,0 +1,208 @@
+package net.maizegenetics.phgv2.pathing
+
+import biokotlin.util.bufferedReader
+import biokotlin.util.bufferedWriter
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.options.validate
+import htsjdk.variant.vcf.VCFFileReader
+import net.maizegenetics.phgv2.utils.Position
+import net.maizegenetics.phgv2.utils.parseALTHeader
+import org.apache.logging.log4j.LogManager
+import java.io.File
+
+/**
+ * This class provides metrics on the h.vcf file created from the imputation pipeline.
+ * It takes an imputation h.vcf file as well as the h.vcf file from the sample that was
+ * the parent of the short reads used in the imputation pipeline.  It also takes a bed file
+ * that was used to create the ranges for the tiledb data storage.  It will output a file
+ * that, for every range in the bed file, displays the haplotype IDs at that range in both
+ * the imputation h.vcf as well as the parent sample h.vcf.  It will also display the read
+ * counts for each haplotype ID to allow comparisons between the sample chosend at a given
+ * range vs the read counts for the parent sample at that range.
+ *
+ * In addition, there are columns to note if the range was present in the imputation h.vcf
+ * or in the parent sample h.vcf file, and a column that indicates if the 2 haplotypes (parent
+ * and imputation) are the same.
+ *
+ * The output file will be tab-delimited and will be written to the output file specified
+ * by the user.
+ */
+
+class ImputationMetrics : CliktCommand(help = "Impute best path(s) using read mappings.")   {
+    private val myLogger = LogManager.getLogger(ImputationMetrics::class.java)
+    val sampleHvcf by option(help = "Path to hvcf file loaded to tiledb for the Parent sample of your short reads. For example: If your reads are from P39, this would be the P39.h.vcf.Required parameter.")
```

Comment:
With Terry's new changes does the Required parameter in the help need to be there?  Or does the help already show it?

Reply:
No it will say required automatically

Reply:
To clarify - you would like me to remove "Required parameter" from each of the inline parameter descriptions?

Reply:
yes I think the 'Required parameter." portion of the help is not needed anymore.